src/main/java/org/candlepin/subscriptions/tally/MetricUsageCollector.java

```diff
@@ -528,11 +534,14 @@ private AccountUsageCalculation loadHourlyAccountCalculation(Event event) {
               .forEach(
                   entry -> {
                     var measurementKey = entry.getKey();
-                    calc.addUsage(
-                        usageKey,
-                        measurementKey.getMeasurementType(),
-                        MetricId.fromString(measurementKey.getMetricId()),
-                        entry.getValue());
+                    var metricId = MetricId.fromString(measurementKey.getMetricId());
+                    if (isMetricSupportedByProduct(metricId, usageKey.getProductId())) {
```

Comment:
Same as above, think we should log when we skip a metric.

Reply:
Added